plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -2000,5 +2001,41 @@ var _ = Describe("validator", func() {
 				Expect(err).To(HaveOccurred())
 			})
 		})
+
+		Context("shoot deletion", func() {
+			var (
+				shootStore             cache.Store
+				intCoreInformerFactory coreinformers.SharedInformerFactory
+			)
+
+			BeforeEach(func() {
+				intCoreInformerFactory = coreinformers.NewSharedInformerFactory(nil, 0)
+				shootStore = intCoreInformerFactory.Core().InternalVersion().Shoots().Informer().GetStore()
+				Expect(coreInformerFactory.Core().InternalVersion().CloudProfiles().Informer().GetStore().Add(&cloudProfile)).To(Succeed())
+				Expect(coreInformerFactory.Core().InternalVersion().Seeds().Informer().GetStore().Add(&seed)).To(Succeed())
+				Expect(coreInformerFactory.Core().InternalVersion().Projects().Informer().GetStore().Add(&project)).To(Succeed())
+			})
+
+			DescribeTable("DeleteShootInMigration", func(lastOperationType core.LastOperationType, matcher types.GomegaMatcher) {
```

Comment:
Just move the func signature on new line and also indent its body with one more tab.
```suggestion
			DescribeTable("DeleteShootInMigration", 
				func(lastOperationType core.LastOperationType, matcher types.GomegaMatcher) {
```